doc/src/appendices/site-user-config-ref.rst

```diff
@@ -180,14 +180,14 @@ Documentation locations for the ``cylc doc`` command and gcylc
 Help menus.
 
 [documentation] ``->`` [[online]]
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 URL of the online cylc documentation.
 
 [documentation] ``->`` [[local]]
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Path where the Cylc documentation will appear if built locally.
+Path where the cylc documentation will appear if built locally.
```

Comment:
Isn't that Cylc, as in Cylc project? Just out of curiosity, because I normally use Cylc for project name, and cylc for command name.

Reply:
True, I agree it should really be the capitalised form in general, so I originally went in to change another line nearby, but then I saw that all nearby content used the lower-case variant. I then did a quick skim through a sample of the rest of the docs, where the lower case seemed to be used.

So I made this change for consistency. Sorry, I probably should have clarified that in the opening comment. (As an aside I am not *quite* pedantic enough to go through & via grepping & checking blitz the entire docs to convert non-command etc. references from ``cylc`` to ``Cylc``, even if that is preferred!)

Reply:
Not a problem. Thanks for clarifying Sadie!

Reply:
I originally wrote "Cylc" (the project) as "cylc" too, but was eventually persuaded by a manager that it stands out more in documents if capitalized. But there's still a mix of cases in the user guide (my fault!).